schema/_value.go

```diff
@@ -0,0 +1,37 @@
+package schema
+
+import (
+	"fmt"
+	"strings"
+)
+
+// in the future.. see: https://github.com/webrpc/webrpc/issues/122
```

Comment:
I think its fine to keep this file in the source for the future, but leave it _value.go so its omitted